chainer/functions/__init__.py

```diff
@@ -170,12 +171,16 @@
 det = det.det
 Exp = exponential.Exp
 exp = exponential.exp
+Expm1 = exponential_m1.Expm1
+expm1 = exponential_m1.expm1
 Identity = identity.Identity
 identity = identity.identity
 Inv = inv.Inv
 inv = inv.inv
 Log = exponential.Log
 log = exponential.log
+Log1p = exponential_m1.Log1p
+log1p = exponential_m1.log1p
```

Comment:
`m` in `exponential_m1` stand for `minus` (I guess). So I think this package name is not appropriate as a location of `log1p` function.